### DISCUSSION TITLE
How to export SMTP Log?

### DISCUSSION THREAD

**Discussion by niels-przybilla** [postalserver/postal#3242] [Help with using Postal]
Hello Postal Community,

i see I can watch logs with postal logs smtp.

But how can I access the file or export the SMTP Log ?

THX and best wishes

Niels

**Comment by willpower232** [comment]
You will have to configure the logging from the docker compose file and restart Postal to apply.

The following may be helpful

https://docs.docker.com/reference/compose-file/services/#logging
https://docs.docker.com/engine/logging/configure/
